src/nft/pages/collection/index.tsx

```diff
@@ -106,12 +105,16 @@ const Collection = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
 
+  if (loading) return <CollectionPageSkeleton />
+
   const toggleActivity = () => {
     isActivityToggled
       ? navigate(`/nfts/collection/${contractAddress}`)
       : navigate(`/nfts/collection/${contractAddress}/activity`)
   }
 
+  const EMPTY_TRAIT_OBJ = {}
```

Comment:
probably better to define outside of react components so it's not a new object every render